libs/barista-components/table/src/table.ts

```diff
@@ -321,4 +332,41 @@ export class DtTable<T> extends _DtTableBase<T> implements OnDestroy {
 
   /** CSS class added to any row or cell that has sticky positioning applied. */
   protected stickyCssClass = 'dt-table-sticky';
+
+  exportCSV(): void {
```

Comment:
Missing JSDoc, also this can be considered an internal function. Would need to mark it as such in the JSDoc and prefix with an _

```suggestion
/** @internal Converts the datasources data to csv format, adds it to a blob and initiates a download
  _exportCSV(): void {
```